native/pmd.c

```diff
@@ -16,25 +67,22 @@ static const struct rte_eth_conf default_eth_conf = {
             .mq_mode        = ETH_MQ_RX_RSS, /* Use RSS without DCB or VMDQ */
             .max_rx_pkt_len = 0,             /* valid only if jumbo is on */
             .split_hdr_size = 0,             /* valid only if HS is on */
-            .header_split   = 0,             /* Header Split off */
-            .hw_ip_checksum = HW_RXCSUM,     /* IP checksum offload */
-            .hw_vlan_filter = 0,             /* VLAN filtering */
-            .hw_vlan_strip  = 0,             /* VLAN strip */
-            .hw_vlan_extend = 0,             /* Extended VLAN */
-            .jumbo_frame    = 0,             /* Jumbo Frame support */
-            .hw_strip_crc   = 1,             /* CRC stripped by hardware */
+            .offloads       = 0,             /* offloads api */
         },
     .txmode =
         {
-            .mq_mode = ETH_MQ_TX_NONE, /* Disable DCB and VMDQ */
+            .mq_mode                 = ETH_MQ_TX_NONE, /* Disable DCB and VMDQ */
+            .hw_vlan_reject_tagged   = 0,
+            .hw_vlan_reject_untagged = 0,
+            .hw_vlan_insert_pvid     = 0,
+            .offloads                = 0,
         },
-    /* FIXME: Find supported RSS hashes from rte_eth_dev_get_info */
     .rx_adv_conf.rss_conf =
         {
             .rss_hf  = ETH_RSS_IP | ETH_RSS_UDP | ETH_RSS_TCP | ETH_RSS_SCTP,
             .rss_key = NULL,
         },
-    /* No flow director */
+    // turned flow director off ~ : https://doc.dpdk.org/dts/test_plans/fdir_test_plan.html
```

Comment:
was off previously, but something we can look at. Additionaly, flow bifurcation may be something to look at as well: https://doc.dpdk.org/guides/howto/flow_bifurcation.html.